collectors/cveorg/collectors.py

```diff
@@ -0,0 +1,328 @@
+import json
+import os
+import re
+from typing import Union
+
+from celery.utils.log import get_task_logger
+from django.conf import settings
+from django.db import transaction
+from django.utils import timezone
+from django.utils.dateparse import parse_datetime
+
+from apps.taskman.constants import JIRA_AUTH_TOKEN
+from collectors.cmd import Cmd
+from collectors.constants import SNIPPET_CREATION_ENABLED, SNIPPET_CREATION_START_DATE
+from collectors.cveorg.constants import CELERY_PVC_PATH
+from collectors.framework.models import Collector
+from collectors.keywords import should_create_snippet
+from collectors.utils import handle_urls
+from osidb.core import set_user_acls
+from osidb.models import FlawCVSS, FlawReference, Snippet
+
+logger = get_task_logger(__name__)
+
+
+class CVEorgCollectorException(Exception):
+    pass
+
+
+class CVEorgCollector(Collector):
+    # Snippet creation is disabled for now
+    snippet_creation_enabled = None
+
+    # When the start date is set to None, all snippets are collected
+    # When set to a datetime object, only snippets created after that date are collected
+    snippet_creation_start_date = SNIPPET_CREATION_START_DATE
+
+    BEGINNING = timezone.datetime(2024, 7, 1, tzinfo=timezone.get_current_timezone())
+
+    REPO_URL = "https://github.com/CVEProject/cvelistV5.git"
+
+    REPO_PATH = f"{CELERY_PVC_PATH}/cvelistV5"
+    # Matches e.g. cves/2023/21xxx/CVE-2023-0001.json, cves/2023/22xxx/CVE-2023-22688.json
+    CVE_PATH = r"cves/(?:19|20)\d{2}/.*/CVE-(?:1999|2\d{3})-(?!0{4})(?:0\d{3}|[1-9]\d{3,}).json$"
+
+    # From https://cveproject.github.io/cve-schema/schema/docs/#oneOf_i0_containers_cna_descriptions_contains_lang
+    EN_LANG = r"^en([_-][A-Za-z]{4})?([_-]([A-Za-z]{2}|[0-9]{3}))?$"
+
+    def __init__(self) -> None:
+        """
+        Initiate collector.
```

Comment:
`Initiate` -> `Initialize`

Reply:
Good catch. I am going to remove it as it does not have any extra value.